docs/TuxStores.md

```diff
@@ -0,0 +1,172 @@
+
+```
+    var listenerCallback = function () {
+      this.setState({ messages: MessageStore.all() });
+    };
+    var newChangeEvent = 'NEW_CHANGE_EVENT';
+    store.addChangeListener(listenerCallback, newChangeEvent);
+```
+
+
+###<a id="store-removeChangeListener"></a>Method `store.removeChangeListener` - type: FUNCTION [#](#store-removeChangeListener)
+A store instance is able to invoke the `removeChangeListener` method, which will remove specified event type string and callback combination from the store. When removing a change listener ensure that the callback and CHANGE_EVENT input parameters are the same values that were submitted when adding the change listener. The change event can be either the default event string `CHANGE` or one provided by the user.
+
+```
+    store.removeChangeListener(callback, CHANGE_EVENT);
+```
+
+#####<a id="removeChangeListener-callback"></a>Parameter - `callback` - type: FUNCTION - required [#](#removeChangeListener-callback)
+The callback function that will be removed from the stores set of listeners when the `removeChangeListener` is invoked.
+
+```
+    var listenerCallback = function () {
+      this.setState({ messages: MessageStore.all() });
+    };
+    store.removeChangeListener(listenerCallback);
+```
+
+#####<a id="removeChangeListener-CHANGE_EVENT"></a>Parameter - `CHANGE_EVENT` - type: STRING - optional [#](#removeChangeListener-CHANGE_EVENT)
+Change event type string that will be used to remove the listener from the store. If no `CHANGE_EVENT` is specified the default value of `CHANGE` will be used.
+
+```
+    var listenerCallback = function () {
+      this.setState({ messages: MessageStore.all() });
+    };
+    var newChangeEvent = 'NEW_CHANGE_EVENT';
+    store.removeChangeListener(listenerCallback, newChangeEvent);
+```
+
+##<a id="TuxStores-Complete-Example"></a>TuxStores Complete Example [#](#TuxStores-Complete-Example)
+Taking everything we have learned lets look at a more realistic example of using TuxStores.
+
+```
+    // messageStore.js
+    var Stores = require('tux/Stores');
+
+    module.exports = Stores.createStore({
+
+      _messages: [],
+
+      allMessages: function () {
+        return this._messages;
+      },
+
+      onGet: function (data) {
+        this._messages = data.messages;
+        this.emitChange();
+      },
+
+      onCreate: function (message) {
+        this._messages.push(message);
+        this.emitChange();
+      },
+
+      onDelete: function (message) {
+        this._messages.splice(message.id, 1);
+        this.emitChange();
+      }
+    });
+
+
+    // In addition to the key/value pairs on the object that was passed into the createStore method, the object that was returned and is being exported has the emitChange, addChangeListener, and removeChangeListener methods as well.
```

Comment:
Remove this comment or change it to three console.assert statements:

`console.assert(messageStore.hasOwnProperty('emitChange');`